conformance/tests/httproute-timeout-backend-request.go

```diff
@@ -59,6 +59,10 @@ var HTTPRouteTimeoutBackendRequest = suite.ConformanceTest{
 				Request:   http.Request{Path: "/backend-timeout?delay=1s"},
 				Response:  http.Response{StatusCode: 504},
 				Namespace: ns,
+			}, {
+				Request:   http.Request{Path: "/disable-backend-timeout?delay=2s"},
```

Comment:
nit: maybe make the delay just 1s (or even less)? That will also prove that "0s" == no timeout, but speeds up the tests.

Reply:
> I also wanted the timeout to be > than the delay from the other test case - to ensure there is isolation between the rules.

I don't see how 2s does that any better than 1s ... anything greater than 500ms will do that.

I still would prefer the timeout be reduced from 2s -> 1s, to eliminate an extra couple of seconds of unnecessary delay (2 tests x 1s) that doesn't prove anything extra. I think we should try to minimize unnecessary delays wherever we can because they do add up.

Reply:
1s sounds good